mscore/preferenceslistwidget.h

```diff
@@ -30,15 +30,17 @@ namespace Ms {
 //---------------------------------------------------------
 //   PreferenceItem
 //---------------------------------------------------------
-class PreferenceItem : public QTreeWidgetItem, public QObject {
+class PreferenceItem : public QObject, public QTreeWidgetItem {
+
+      Q_OBJECT
 
       QString _name;
 
     protected:
       void save(QVariant value);
 
     public:
-      PreferenceItem();
+      PreferenceItem() {};
```

Comment:
This default constructor is not needed, and in fact is probably a bad idea.

Reply:
Agreed, removed it.